### PR TITLE
README

### PR DESCRIPTION
**Pull request by ForChune**
Following the review, I modified the README.

### CODE REVIEW

**Comment by sharonibejih** [comment]
I looveeee this! 🔥
For me, it's can be merged.
Thanks Fortune!

File: README.md [hunk 1]
```diff
@@ -1,10 +1,22 @@
...
+
+The protests were mostly peaceful, however, videos were littered all over twitter showing the police firing rubber bullets, stun guns and tear gas at protesters and journalists to intimidate them. This shocked many Americans who were for the first time witnessing their police’s heavy-handedness in dealing with peaceful protesters.
+
+In recent years, there have been numerous cases involving the fatal shooting of Black Americans by the police. In majority of these cases, the police officers involved in the shooting were let off with a slap on the wrist leaving the victims’ families unsatisfied with the ruling. 
+
+Based on this, we decided to conduct an analysis of police shooting incidents in America from 2015 to date, to ascertain if the shootings were justified in a bid to help victims and their families seeking justice to get it. To achieve this, we performed clustering technique on our data to discover patterns about police shootings that could help the victims and their families in their lawsuit against the police.
```
Comment (sharonibejih) [inline README.md]: We ended up using a Supervised learning approach. Can you please edit it, and change that line to something like (just a rough clue): To achieve this, we used a supervised learning algorithm to predict the probability of an unjustified shot. Cases with probabilities equal to or higher than 60% are most likely unjust. Hence, this approach could help the victims or their families and governing bodies in their lawsuit against such Police officer(s).

**Review by sharonibejih — COMMENTED** [review]
Sorry Fortune, someone called my attention to something. I really don't mean to draw you back again.
It's the last paragraph under the Motivation. I will highlight it.

Please help edit it. Thanks a lot.

File: README.md (hunk 1 above)
Comment (Jahfaith) [reply README.md]: Weldon @ForChune. It’s a nice READ ME and with the little adjustment spotted by @sharonibejih I think we’d be good to go.

Comment (ForChune) [reply README.md]: Thanks for the review! @sharonibejih @Jahfaith.  I've made some modifications to the second to the last paragraph based on your feedback.

**Review by sharonibejih — COMMENTED** [review]
Thanks Fortune, this is just perfect. 

Please go ahead and merge.

**Review by Jahfaith — APPROVED** [review]

**Pull request merged**